extensions/wikia/UserProfilePageV3/js/UserProfilePage.js

```diff
@@ -156,29 +160,27 @@ var UserProfilePage = {
 							UserProfilePage.bucky.timer.stop('renderLightboxSuccess');
 						});
 					});
-				});
-			}).fail(function (data) {
+				}).fail(function (data) {
 
-				console.log(data);
+					window.console.log(data);
 
-				var message = '',
-					response = '';
+					var message = '',
+						response = '';
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "response" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AS1854)